### ISSUE TITLE
Add. 17,215, fol. 35 | https://bl.vuexistapps.us/manuscript/1410 (Part12)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#714]
### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- Hand 1: The sentence "Fol. 35was a fly-leaf, and contains on the verso a note" appears in Hand 1. The fact that this folio is a fly-leaf was mentioned earlier in the main paragraph of Phys. Des. and the rest of the sentence, indicating that the folio contains a scribal note, is more likely to belong to the content section rather than Hand 1. So, section Hand 1 says nothing related to the hand or script. Wright, however, has "The writing is inelegant, of the xth or xith cent," but perhaps it can not be understood if the same info is true also about this scribal note on fol. 35. So, in case this sentence is about the whole manuscript, including the note on the fly-leaf, it should be inserted in the Hand section of Add. 17,215, fol. 35 as well. 
- Hand 1: Lack of space in "Fol. 35was"
- Hand 1: Lack of period mark at the end of the sentence. 
- Accompanying Material is empty. 

### Categories of Physical Description Problems

- [X] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

**Comment by mikey-tang16** [comment]
As a solution to the previous question, can I simply leave the handnote blank and keep everything else as is?
@wlpotter

**Comment by wlpotter** [comment]
Agreed, keep the hand note empty (but with script of 'syr') for this one
